src/main/java/seedu/address/model/WebLink/WebLink.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.weblink;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a WebLink in the address book.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidWebLink(String)}
+ */
+public class WebLink {
+
+    public static final String MESSAGE_WEB_LINK_CONSTRAINTS = "it should be a web link you want to stalk.";
+    public static final String WEB_LINK_VALIDATION_REGEX = "^\\S+";
```

Comment:
Might want to make this more defined....... for future update.

Reply:
hahaha okayokay

Reply:
okay the regex i set as such cos, really have too many permutations, https://, http, www, or no www. I didnt see a point in setting a restrictive regex.